packages/cohere/src/cohere-chat-language-model.test.ts

```diff
@@ -45,6 +47,14 @@ describe('doGenerate', () => {
     generation_id?: string;
     headers?: Record<string, string>;
   }) {
+    const content = [];
+    if (typeof text === 'string') {
+      content.push({ type: 'text', text });
+    }
+    if (typeof thinking === 'string') {
+      content.push({ type: 'thinking', thinking });
+    }
```

Comment:
i would have passed content[] instead and removed txt to match their api more closesly

Reply:
you mean this?

```diff
-    text?: string | null;
-    thinking?: string | null;
+   content: Array<{type:'text', text: string} | {type:'thinking', thinking: string}>;
```

Reply:
ah yes just saw your other comments